### ISSUE TITLE
desktop Notifications for electrum

### ISSUE THREAD

**Issue by bitsam01** [blockchain/blockchain-wallet-v4-frontend#1071]
# 🐛 Bug Report 🐛

## Description
i do receive bitcoin to electrum but dont know when it arrived please help develop a notification

## Expected Behavior
inapp side notification

## Reproduction Steps
Enter detailed steps to reproduce the issue.

## Screenshots/Logs
If available, please provide any relevant screenshots or logs.

## Additional Information
Add any addition information that might be useful to our team.

**Comment by plondon** [comment]
Like these? https://developers.google.com/web/fundamentals/push-notifications/

**Comment by bitsam01** [comment]
Yes, it recalls a transaction been done when no available or been received

**Comment by TioRha** [comment]
Like these

**Comment by bitsam01** [comment]
I mean if i received a payment of bitcoin and was not online anytime i came online before i access my email or app there should be a app notification that blinks up that payment have been received

**Comment by essubroto** [comment]
@essubroto : Nah cara pemberian label itu bagaiamana?

**Comment by essubroto** [comment]
Oke,

Pada tanggal Rab, 26 Jun 2019 20:45, Santi Dwi Setiawan <
notifications@github.com> menulis:

**Comment by dimon00048** [comment]
здравствуйте, подскажите пожалуйста можно вернуть неподтвержденную (0/3)транзакцию блокчайна?

**Comment by Wiphawee112** [comment]
https://developers.google.com/web/fundamentals/push-notifications/

**Comment by princeanndy** [comment]
This website is just a stupid side I have ever seen. If you take live support it will take a log time with queue and when ever you number will come they won't response you. just disconnect. I I need good at them. They are dealing with  money for money but they don't have support in case you need it. I don't recement  people to use this app. or their website. they are just like stupid.

**Comment by aivarsmarti** [comment]
Bdd

**Comment by Timur95S** [comment]
>

**Comment by Jefferysteve3111** [comment]
> # 🐛 Bug Report 🐛
> ## Description
> i do receive bitcoin to electrum but dont know when it arrived please help develop a notification
> 
> ## Expected Behavior
> inapp side notification
> 
> ## Reproduction Steps
> Enter detailed steps to reproduce the issue.
> 
> ## Screenshots/Logs
> If available, please provide any relevant screenshots or logs.
> 
> ## Additional Information
> Add any addition information that might be useful to our team.

Talk to Mr. Steve on WhatsApp and he will help you. ‪+1 (707) 389‑5639‬

**Comment by Chris6900** [comment]
I’m sorry everyone it’s my original block chain and I had over 1,000,000 bucks worth of bit coin and I developed a block chain to send my funds to an account not knowing what I was doin and I’m afraid I’ve lost it all
